console/src/components/Common/CollapsibleToggle/CollapsibleToggle.tsx

```diff
@@ -9,8 +9,25 @@ import React from 'react';
  *    `children, react-element`: The content which needs to be toggled
  */
 
-class CollapsibleToggle extends React.Component {
-  constructor(props) {
+interface CollapsibleToggleProps {
+  title: string;
+  isOpen: boolean;
+  toggleHandler?: () => void;
+  testId: string;
+  children: React.ReactNode;
```

Comment:
Typings for `children` will already be there as we are extending `React.Component`.
```suggestion
```

Reply:
sorry i didn't load vs-code from console folder so eslint rules weren't loaded, i corrected and solved those issues now.